src/aws/osml/tile_server/services/token.py

```diff
@@ -22,6 +24,8 @@ def initialize_token_key() -> None:
     :return: None
     """
     try:
+        if not os.path.exists(f"/{ServerConfig.efs_mount_name}"):
+            os.makedirs(f"/{ServerConfig.efs_mount_name}")
```

Comment:
Is this a bug fix? If so try not to hide things like this in the middle of a refactor PR.

Reply:
I don't know if we have run into this as an actual bug but it was something I notices and a 2 line change to prevent a bug.  I can back it out if you prefer.